lib/update_school_data.rb

```diff
@@ -29,6 +29,7 @@ def convert_to_school(row)
     school
   end
 
+  # rubocop:disable Metrics/AbcSize
```

Comment:
Not an issue but I think you can remove the need to disable this if you move all the new stuff into a separate method and call the method within set properties.

```
def set_properties(school, row)
    school.name = row['EstablishmentName']
    school.address = row['Street']
    school.locality = row['Locality'].presence
    school.address3 = row['Address3'].presence
    school.town = row['Town']
    school.county = row['County (name)'].presence
    school.postcode = row['Postcode']
    school.local_authority = row['LA (name)']
    school.minimum_age = row['StatutoryLowAge']
    school.maximum_age = row['StatutoryHighAge']
    school.easting = row['Easting']
    school.northing = row['Northing']
    school.url = valid_website(row['SchoolWebsite'])
    school.phase = row['PhaseOfEducation (code)'].to_i
    set_school_key_info(school, row)
  end

  def set_school_key_info(school, row)
    school.status = row['EstablishmentStatus (name)']
    school.trust_name = row['Trusts (name)']
    school.number_of_pupils = row['NumberOfPupils']
    school.head_title = row['HeadTitle (name)']
    school.head_first_name = row['HeadFirstName']
    school.head_last_name = row['HeadLastName']
    school.religious_character = row['ReligiousCharacter (name)']
    school.rsc_region = row['RSCRegion (name)']
    school.telephone = row['TelephoneNum']
    school.open_date = row['OpenDate']
    school.close_date = row['CloseDate']
    school.last_ofsted_inspection_date = row['OfstedLastInsp']
    school.oftsed_rating = row['OfstedRating (name)']
  end
```

Reply:
Not all those columns are needed for the key info snippets, we decided to put it in one method based on `export_vacancy_records_to_big_query.rb` and just disable to metric in order to keep it all in one place

Reply:
These long lists of import attributes are difficult to read and maintain and fragile.

Can you please replace the hardcoded list of keys/attributes with an abstract method that maps keys of known format to symbols suitable for importing and have the migration follow the same schema.  Please make the attribute names follow the names used by the API (so `EstablishmentName` becomes `school.establishment_name` - you can alias a `school.name` public method to it to expose that for internal use).